Library/Homebrew/livecheck_strategy.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+module LivecheckStrategy
```

Comment:
```suggestion
module Homebrew::Livecheck::Strategy
```

Reply:
If strategies are moving to `livecheck/strategies` (e.g., `livecheck/strategies/apache`) should `livecheck_strategy.rb` be moved into the `livecheck` folder as well? If so, does the file need to be renamed?

Reply:
Yeh, I think so. `livecheck/strategy.rb` or similar may make sense.